decisions/wc_order_decisions.txt

```diff
@@ -250,8 +250,46 @@ decisions = {
 			factor = 1
 		}
 	}
-}
 	
+	create_blood_knights_order = {
+		ai_check_interval = 12
+		is_high_prio = yes
+		
+		only_playable = yes
+		
+		potential = {
+			religion = cult_of_sunwell
+			OR = {
+				has_landed_title = e_quelthalas
+				has_landed_title = k_quelthalas
+			}
+			is_high_elf_trigger = yes
+			
+			d_blood_knights_order = { has_holder = no }
+		}
```

Comment:
I remember there was culture condition...
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81938972-c5292680-9606-11ea-8ad5-94e43c66ea9e.png)

</details>

Reply:
![grafik](https://user-images.githubusercontent.com/47716948/81943161-52ff1480-95fb-11ea-9d15-dea77fd88803.png)
It was trait-locked but you suggested using trigger

Reply:
Even before trait-lock there was culture condition, but you replaced it with trigger

Reply:
> It was trait-locked but you suggested using trigger
It's still race-locked, I'm an elf with Amani culture.

My bad. Meant `is_high_elf_culture_trigger`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81944101-4c799880-960d-11ea-841e-9396ae71c961.png)

</details>